fv3core/external/shared

```diff
@@ -0,0 +1 @@
+../../shared
```

Comment:
Yes, good catch

Reply:
Since physics does not have a docker file now, just adding the `stencils` mount in the makefile. In the future, we would want to have the same external that `fv3core` has.